docs/diracdoctools/Utilities.py

```diff
@@ -107,8 +107,11 @@ def runCommand(command):
     result = subprocess.check_output(shlex.split(command), stderr=subprocess.STDOUT,
                                      universal_newlines=True)
     if 'NOTICE:' in result:
-      LOG.warn('NOTICE in output for: %s', command)
-      return ''
+      lines = []
+      LOG.warn('NOTICE in output for: %s; cleaning output from datestamp..', command)
+      for l in result.split('\n'):
+        lines.append(re.sub(r"^.*NOTICE: ", "", l))
+      result = "\n".join(lines) if len(lines) > 2 else ''
```

Comment:
why `len(lines) > 2`, why not just
```suggestion
      result = "\n".join(lines)
```

Reply:
I think if the result of output "command --help" is a few lines, it means that something went wrong, no?

the minimum help output is exactly more than two lines

Reply:
Ok makes sense, can you add this as a comment?
```suggestion
      # if the output is less than 3 lines something went wrong
      result = "\n".join(lines) if len(lines) > 2 else ''
```

Reply:
added